packages/core/src/operator/operator-runtime/operator-v1/bootOperatorRuntime.ts

```diff
@@ -22,7 +22,7 @@ export const bootOperatorRuntime_V1 = async (
         const openChallenge = await retry(() => getLatestChallengeFromGraph());
         // Calculate the latest challenge we should load from the graph
         const latestClaimableChallenge = Number(openChallenge.challengeNumber) <= MAX_CHALLENGE_CLAIM_AMOUNT ? 1 : Number(openChallenge.challengeNumber) - MAX_CHALLENGE_CLAIM_AMOUNT;
-    
+      
```

Comment:
Remember the merge conflict which had an empty line, this will lead to something like that, just fyi, you don't need to do anything here.